Automattic/wp-calypso
# Revert single-column plans grid to v2 carousel and exclude US and CA from AB test

#### Changes proposed in this Pull Request

This PR reverts https://github.com/Automattic/wp-calypso/pull/44299 to restore the plans grid as it was in v2. 
 
#### Testing instructions

Visit [`/new`](https://calypso.live/new?branch=revert/accordian-plans-grid) and see if the plans grid looks and works as it should.
